app/controllers/requests_controller.rb

```diff
@@ -78,7 +78,7 @@ def cdl_waitlist_position
   def handle_pickup_change_request
     change_pickup_response = ils_client.change_pickup_library(*change_pickup_params)
     case change_pickup_response.status
-    when 200
+    when 200, 204
```

Comment:
Do we need both of these? When does it return one over the other?

Reply:
Yes, the folio response is a 204. I can leave a comment about this.